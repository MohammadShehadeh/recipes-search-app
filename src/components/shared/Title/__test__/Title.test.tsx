import renderer from 'react-test-renderer';

import { Title } from '..';

describe('Title', () => {
  it('should render "primary" to match snapshot', () => {
    const tree = renderer.create(<Title variant="primary">Primary Title</Title>).toJSON();
    expect(tree).toMatchSnapshot();
  });

  it('should render "secondary" to match snapshot', () => {
    const tree = renderer.create(<Title variant="secondary">Secondary Title</Title>).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
