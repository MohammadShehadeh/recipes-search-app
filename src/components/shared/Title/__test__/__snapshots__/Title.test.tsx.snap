// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Title should render "primary" to match snapshot 1`] = `
<h2
  className="title primary"
>
  Primary Title
</h2>
`;

exports[`Title should render "secondary" to match snapshot 1`] = `
<h2
  className="title secondary"
>
  Secondary Title
</h2>
`;
